eng/download-workloads.ps1

```diff
@@ -0,0 +1,73 @@
+param ([Parameter(Mandatory=$true)] [string] $workloadPath, [SecureString] $gitHubPat, [SecureString] $azDOPat, [string] $workloadListJson = '', [bool] $usePreComponents = $false)
```

Comment:
same as above. Comment the parameters and their common format

Reply:
Added a description for the script, descriptions for the parameters, and example values for the parameters (as necessary).